src/config/setupVars.c

```diff
@@ -137,12 +137,69 @@ static void get_conf_string_array_from_setupVars(const char *key, struct conf_it
 		getSetupVarsArray(array);
 		for (unsigned int i = 0; i < setupVarsElements; ++i)
 		{
-			// Add string to our JSON array
-			cJSON *item = cJSON_CreateString(setupVarsArray[i]);
-			cJSON_AddItemToArray(conf_item->v.json, item);
-
-			log_debug(DEBUG_CONFIG, "setupVars.conf:%s -> Setting %s[%u] = %s\n",
-			          key, conf_item->k, i, item->valuestring);
+			// Convert to regex if requested
+			if(convert_to_regex)
```

Comment:
The idea was to have the function being more generic in case we need it again int he future and forgot that it does this automagical conversion. I removed the boolean and, instead, renamed the function to make this clear. We probably won't need the non-regex version.